default.nix

```diff
@@ -1,35 +1,41 @@
-{ rocksDBVersion ? "6.29.3" }:
+{ versions ? import ./nix/versions.nix
+, release ? true
+, name ? "aleph-node"
+, crates ? { "aleph-node" = []; }
+, runTests ? false
+, singleStep ? false
+, rustflags ? "-C target-cpu=native"
```

Comment:
Does this come from any particular place? What `RUSTFLAGS` we use now?

Reply:
github pipelines use `-C target-cpu=x86-64-v2` - I saw once `ILLEGAL INSTRUCTION` and got pretty scared.

Reply:
As discussed, let's keep this value as `generic`